.github/global.yml

```
@@ -230,3 +230,17 @@
   description: related to producing reproducible results
   color: "86C579"
   aliases: [reproducibility]
+
+# Documentation
+- name: documentation::tutorials
+  description: related to tutorials (learning-oriented)
+  color: "31A354"
```

This color is too similar to the ones used for:

- (#43b049): https://github.com/conda/infra/labels/good-first-issue https://github.com/conda/infra/labels/help-wanted https://github.com/conda/infra/labels/mitigated
- (#43b04b): https://github.com/conda/infra/labels/severity%3A%3A4

I suggest we use a soft blue:

```suggestion
  color: "77b2dc"
```